### ISSUE TITLE
There is no "Open All" button. If you add multiple .torrent files, you must confirm each one in a separate window.

### ISSUE THREAD

**Issue by moninah850** [transmission/transmission#6445]
### What is the issue?

There is no "Open All" button. If you are adding multiple .torrent files, you will need to confirm each one in a separate window. If the number of files to be added is, for example, 100, then you need to click the “Open” button 100 times in a separate window.

### Which application of Transmission?

Qt app on Windows

### Which version of Transmission?

4.0.5

**Comment by tearfur** [comment]
You can use the watch folder instead to add torrents in bulk.

**Comment by killemov** [comment]
@moninah850 Yes that is annoying. And yes, you could the watch folder in this case as it runs on your local client anyway. But I use [Shift](https://github.com/killemov/Shift). Drag (from almost anywhere), drop, done. (Disclaimer: I made that.)

**Comment by moninah850** [comment]
The easiest and most reliable way is to simply add an "Open All" button so that you can add all .torrents with the click of a single button. This is implemented in all programs.

**Comment by killemov** [comment]
> I select several .torrent files from the watch folder

No! You put torrent files IN the watch folder to be processed by Transmission automatically.

> The most reliable way is to provide the ability to add all selected torrent files with one click (no matter how many .torrent files are selected). This is implemented in many programs.

That appears to be technically impossible unless you count drag+drop as one click. 2 clicks sure, but 1? On Windows? Where you must double-click on almost anything?

**Comment by moninah850** [comment]
> That appears to be technically impossible unless you count drag+drop as one click. 2 clicks sure, but 1? On Windows? Where you must double-click on almost anything?

I mean this:

“Ctrl+O” → Select several .torrent files → You need to click the “Open” button as many times as .torrent files are selected. For example, in uTorrent you select any number of files and the “Open” button needs to be clicked only once and all selected .torrent files will be added.

**Comment by tearfur** [comment]
> “Ctrl+O” → Select several .torrent files → You need to click the “Open” button as many times as .torrent files are selected. For example, in uTorrent you select more than 20 .torrent files and the “Open” button needs to be clicked only once and all selected .torrent files will be added.

Sounds reasonable to me. 👍🏻

**Comment by axu2** [comment]
> So, I select several .torrent files from the watch folder, but after that I need to click "Open" for each torrent.

I added 40 .torrent files to my watched folder, which caused 40 windows to show up. I just held the enter button to click open on all them quickly.
